app-5-rdf-serializer/src/test/java/org/geovistory/toolbox/streams/rdf/processors/project/ProjectCustomRdfsLabelsTest.java

```diff
@@ -0,0 +1,257 @@
+package org.geovistory.toolbox.streams.rdf.processors.project;
+
+
+import io.debezium.data.geometry.Geography;
+import org.apache.kafka.streams.StreamsConfig;
+import org.apache.kafka.streams.TestInputTopic;
+import org.apache.kafka.streams.TestOutputTopic;
+import org.apache.kafka.streams.TopologyTestDriver;
+import org.geovistory.toolbox.streams.avro.*;
+import org.geovistory.toolbox.streams.lib.GeoUtils;
+import org.geovistory.toolbox.streams.rdf.*;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Properties;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ProjectCustomRdfsLabelsTest {
+
+    private static final String SCHEMA_REGISTRY_SCOPE = ProjectCustomRdfsLabelsTest.class.getName();
+    private static final String MOCK_SCHEMA_REGISTRY_URL = "mock://" + SCHEMA_REGISTRY_SCOPE;
+    private TopologyTestDriver testDriver;
+    private TestInputTopic<ProjectRdfKey, ProjectRdfValue> projectTopic;
```

Comment:
The input stream is KStream<dev.projects.project.Key, dev.projects.project.Value>

Reply:
Done